appserver/persistence/entitybean-container/src/main/java/org/glassfish/persistence/ejb/entitybean/container/EntityContainer.java

```diff
@@ -3018,7 +2991,7 @@ public int getMaxCacheSize() {
 	    return this.confMaxCacheSize;
 	}
 
-    public void appendStats(StringBuffer sbuf) {
+    public void appendStats(StringBuilder sbuf) {
```

Comment:
This method is not used anywhere. If it was it would depend on if the `StringBuilder` passed into it is shared between threads. Possibly being a member of some owning class. 

A StringBuilder param in this method would suggest anyone who will make use of it in the future  that the passed in StringBuilder may not be a shared one.